kaolin/datasets/shrec.py

```diff
@@ -35,70 +37,123 @@ class SHREC16(Dataset):
         categories (list): List of categories to load (each class is
             specified as a string, and must be a valid `SHREC16`
             category).
-        mode (str, choices=['train', 'test']): Whether to load the
-            'train' split or the 'test' split
+        train (optional, bool): If True, return the train split, else return the test
```

Comment:
It is `optional` because we have a default value.

Reply:
I think there is a misunderstand on the definition of "optional" the type `Optional` (usually used for type hints) means that it can be `None` (https://docs.python.org/3/library/typing.html#typing.Optional).

Having a default value is already indicated by `(default: True)`